content/spin/writing-apps.md

```diff
@@ -388,8 +388,10 @@ channel = "messages"
 ```
 
 ## Next Steps
+
 - Learn about how to [build your Spin application code](build)
 - Try [running your application locally](running-apps)
+- Learn how the `spin watch` [command](https://developer.fermyon.com/common/cli-reference#watch) can [monitor changes in your application](https://developer.fermyon.com/spin/running-apps#monitoring-applications-for-changes)
```

Comment:
Sorry I missed this before.  As this page already links to the running apps page as a 'next steps', I would not call out `spin watch` specially.

(As an aside, we might want to have a philosophy on a good number of 'next steps' to have - I am not sure how many would create a sense of "too many choices, just tell me where to go next" if you see what I mean?)

Reply:
Thanks, I just added that part recently. I have removed the link to watch; as you say, we link to that page already.